src/test/java/com/cloudant/tests/SslAuthenticationTest.java

```diff
@@ -77,11 +78,10 @@ public void localSslAuthenticationDisabled() throws Exception {
 
         server.await();
 
-        ConnectOptions connectionOptions = new ConnectOptions();
-        connectionOptions.setSSLAuthenticationDisabled(true);
+        dbClient = ClientBuilder.url(new URL(server.getUrl()))
```

Comment:
`SimpleHttpsServer extends SimpleHttpServer` so this (and the other tests in this class) can use `CloudantClientHelper.newSimpleHttpServerClient(server)` as well can't they?